.github/pull_request_template.md

```diff
@@ -1,28 +1,24 @@
-## Pull Request Testing ##
-
-- [ ] Describe testing already performed for this Pull Request:</br>
-
-- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>
-
-- [ ] Has the code been checked to ensure that no errors occur during the execution? **[Yes or No]**
-
-- [ ] Do these updates/additions include sufficient testing updates? **[Yes or No]**
-
-- [ ] Please complete this pull request review by **[Fill in date]**.</br>
-
-## Pull Request Checklist ##
-
-- [ ] Review the source issue metadata (required labels, projects, and milestone).
-- [ ] Complete the PR description above.
-- [ ] Ensure the PR title matches the feature branch name.
-- [ ] Check the following:
-- [ ]  Instructions provided on how to run
-- [ ]  Developer's name is replaced by ${user} where necessary throughout the code
-- [ ]  Check that the ecf file has all the proper definitions of variables
-- [ ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
-- [ ]  Check to see that the output directory structure is followed
-- [ ]  Be sure that you are not using MET utilities outside the METplus wrapper structure
-
-- [ ] After submitting the PR, select **Development** issue with the original issue number.
-- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
-- [ ] Close the linked issue.
+<b>Note to developers: You must use this PR template!</b>
+
+## Description of Changes
+
+> Please include a summary of the changes and the related GitHub issue(s). Please also include relevant motivation and context.
+
+## Developer Questions and Checklist
+* Is this a high priorty PR? If so, why and is there a date it needs to be merged by?
+* Do you have any planned upcoming annual leave/PTO?
+* Are there any changes needed for when the jobs are supposed to run?
```

Comment:
Who determines whether a PR is "high priority"?  Do we have a definition of high priority?

Reply:
No concrete definition but would be things like bug fixes for ops, correct failures happening in the parallel, etc.